backend/src/main/java/de/samuelgesang/backend/sites/SiteService.java

```diff
@@ -38,12 +39,38 @@ public Optional<Site> getSiteById(String id) {
         return siteRepository.findById(id);
     }
 
-    public Site createSite(Site site) {
+    public Site createSite(SiteCreateDTO siteCreateDTO) {
+        Site site = new Site();
+        site.setName(siteCreateDTO.getName());
+        site.setBaseURL(siteCreateDTO.getBaseURL());
+        site.setSitemap(siteCreateDTO.getSitemap());
+        site.setUserId(siteCreateDTO.getUserId());
+        site.setCrawlSchedule(siteCreateDTO.getCrawlSchedule());
+        site.setEmail(siteCreateDTO.getEmail());
+        site.setCrawlIds(new ArrayList<>());
+
         String favicon = extractFavicon(site.getBaseURL());
         site.setFavicon(favicon);
+
         return siteRepository.save(site);
     }
 
+    public Site updateSite(SiteUpdateDTO siteUpdateDTO) {
+        Site existingSite = siteRepository.findById(siteUpdateDTO.getId())
```

Comment:
The code for "existingSite" is the same as the old one.
May not bad to make it as a common private method to reduce code.